ncm-ceph/src/main/perl/Ceph/Commands.pm

```diff
@@ -58,7 +58,11 @@ sub run_command_as_ceph {
 # run a command prefixed with ceph and return the output in json format
 sub run_ceph_command {
     my ($self, $command, $msg, %opts) = @_;
-    return $self->run_command([qw(/usr/bin/ceph -f json), @$command], $msg, %opts);
+    my @timeout = ();
```

Comment:
@kwaegema CAF::Process has timeout support, via toutput and texecute methods. use that